eliot-service/eliot/libsentry.py

```diff
@@ -47,7 +45,9 @@ def setup_sentry_logging():
     if _SENTRY_CLIENT:
         handler = SentryHandler(_SENTRY_CLIENT)
         handler.setLevel(logging.ERROR)
-        setup_logging(handler)
```

Comment:
I read through the Raven code for `setup_logging` and all it does is add the handler to the root logger. Raven is deprecated and not getting any more dev time, so I figured I'd save future me some effort and swap that line out for manually adding the handler to the logger.